perses/rjmc/geometry.py

```diff
@@ -193,11 +199,17 @@ def logp_reverse(self, top_proposal, new_coordinates, old_coordinates):
 
                 #propose a bond and calculate its probability
                 bond = self._get_relevant_bond(atom, bond_atom)
-                logp_r = self._bond_logp(internal_coordinates[0], bond, top_proposal.beta)
+                bond_k = bond.type.k*units.kilocalorie_per_mole/units.angstrom**2
+                sigma_r = units.sqrt(1/beta*bond_k)
```

Comment:
This should be `1/(beta*bond_k)`, not `1/beta*bond_k` which is grouped as `(1/beta)*bond_k`